unstructured/partition/lang.py

```diff
@@ -0,0 +1,31 @@
+from typing import List
+
+
+# convert language param (list of strings) into tesseract ocr language format (639-2 variant joined by + signs)
+# NOTE(Shreya): does this arg need a default value?
+def prepare_languages_for_tesseract(languages: List[str] = ["eng"]):
+    # NOTE(Shreya): assumes language codes are in tesseract format (will be updated later)
+    tesseract_languages = ""
+    for lang in languages:
+        tesseract_languages = tesseract_languages + lang + "+"
```

Comment:
List comprehension would be great here `"".join([f"{lang}+" for lang in languages).strip("+")`